apps/picsa-tools/option-tool/src/app/app.component.ts

```diff
@@ -7,4 +10,27 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'picsa-tools-option-tool';
+ 
+  constructor(
+    private matIconRegistry: MatIconRegistry,
+    private domSanitizer: DomSanitizer
+  ) {
+    this.registerIcons();
+  }
+   // register custom icons from the assets/svgs folder for access within the app
+  // icons can be accessed in mat-icon as svgIcon='station_data_${key}'
+  registerIcons() {
+    const icons = {
+      female: 'female',
+      male: 'male'
+    };
+    for (const [key, value] of Object.entries(icons)) {
+      const iconName = `picsa_options_${key}`;
```

Comment:
nice idea to add a custom prefix to the icons, will make them easier to identify in the source code